install-kubeadm.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/bash
+set -o nounset -o errexit
+
+RELEASE="$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)"
```

Comment:
This command and some other was changed slightly when https://github.com/kubernetes/website/pull/6768 was reviewed, I think you should update this script to reflect the merged documentation.

https://kubernetes.io/docs/setup/independent/install-kubeadm/

Reply:
Thanks! I've updated the script.